src/test/java/com/bieganski/ox/PositionCalculatorTest.java

```diff
@@ -33,13 +33,13 @@ public void testCalculateProperValue() {
         Assert.assertEquals(actual, expected);
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class,
-            dataProvider = "badValues")
-    public void testCalculateBadValueShouldThrowException(String userInput) throws IllegalArgumentException {
-        //Given
-        PositionCalculator positionCalculator = new PositionCalculator(new PositionValidator(), 10);
-        //When
-        positionCalculator.calculatePosition(userInput);
-        //Then throws
-    }
+//    @Test(expectedExceptions = IllegalArgumentException.class,
```

Comment:
Kwita na to poproszę.

Reply:
Test został usunięty po zmianie logiki w metodzie parsera. Dodałem informację w migawce.